content/en/platform/corda/4.8/open-source/generating-a-node-dockerform.md

```diff
@@ -11,12 +11,12 @@ menu:
 tags:
 - dockerform
 - node
-title: Dockerform plug-in
+title: Dockerform tasks
 ---
 
-# Dockerform plug-in
+# Dockerform tasks
 
-Corda provides two `gradle` plug-ins: `Cordform` and `Dockerform`. They both allow you to run tasks that automatically generate and configure a local set of nodes for testing and demonstration purposes. This page contains information about the operation of the Dockerform plug-in. Visit the [Cordform](../../../../../en/platform/corda/4.8/open-source/generating-a-node-cordform.md) page for Cordform configuration options.
+Corda's `cordformation` Gradle plugin provides the `Cordform` and `Dockerform` tasks. They both allow you to run tasks that automatically generate and configure a local set of nodes for testing and demonstration purposes. This page contains information about the operation of the Dockerform tasks. Visit the [Cordform](../../../../../en/platform/corda/4.8/open-source/generating-a-node-cordform.md) page for Cordform configuration options.
```

Comment:
"the operation of the Dockerform task" (singular).

Reply:
updated